### PR TITLE
[GH-#255] Adjusted header and block structure fields 

### LINKED ISSUE #255: adjust header / block structure fields
- [x] rename `chain_state_hash` to `root_hash`
- [x] rename `difficulty_target` to `target`
- [x] rename `timestamp` to `time`

### PR DESCRIPTION
**Pull request by Artur64**
Resolves #255 .

### CODE REVIEW

**Comment by thepiwo** [comment]
wait for #263 to merge

File: apps/aecore/lib/aecore/structures/signed_tx.ex [hunk 1]
```diff
@@ -58,8 +58,12 @@ defmodule Aecore.Structures.SignedTx do
     {:ok, %SignedTx{data: tx, signature: signature}}
   end
 
+  def sign_tx(%DataTx{} = tx, priv_key) do
+    {:error, "Wrong key size: #{priv_key}"}
+  end
+
   def sign_tx(tx, _priv_key) do
-    {:error, "Wrong Transaction data structure"}
+    {:error, "Wrong Transaction data structure: #{tx}"}
```
Comment (thepiwo) [inline apps/aecore/lib/aecore/structures/signed_tx.ex]: I think there is an inspect(tx) needed to convert to string

Comment (thepiwo) [reply apps/aecore/lib/aecore/structures/signed_tx.ex]: @Artur64 have you tried, what do you think?

Comment (thepiwo) [reply apps/aecore/lib/aecore/structures/signed_tx.ex]: this was a question, not a change request...

Comment (thepiwo) [reply apps/aecore/lib/aecore/structures/signed_tx.ex]: @Artur64 have you tried, what do you think?

**Comment by thepiwo** [comment]
to merge, first format your code mix format using elixir 1.6.2 and merge master afterwards.

**Comment by Artur64** [comment]
@thepiwo This pull request is no longer needed , I will close #255 issue via resoling pull request for issue #256 conflicts (#256 was based on  #255, so no need to double-resolve conflicts). I think we should close this pull request for now.

**Comment by thepiwo** [comment]
okay, lets close this, as the changes are included in PR #290

**Pull request closed without merging**